backend/src/main/java/gov/cdc/usds/simplereport/service/TestOrderService.java

```diff
@@ -274,30 +330,35 @@ public AddTestResultResponse addTestResult(
       TestOrder savedOrder = _repo.save(order);
       _testEventReportingService.report(savedEvent);
 
-      ArrayList<Boolean> deliveryStatuses = new ArrayList<>();
-
-      PatientLink patientLink = _pls.createPatientLink(savedOrder.getInternalId());
-      if (patientHasDeliveryPreference(savedOrder)) {
-
-        if (smsDeliveryPreference(savedOrder) || smsAndEmailDeliveryPreference(savedOrder)) {
-          boolean smsDeliveryStatus = testResultsDeliveryService.smsTestResults(patientLink);
-          deliveryStatuses.add(smsDeliveryStatus);
-        }
-
-        if (emailDeliveryPreference(savedOrder) || smsAndEmailDeliveryPreference(savedOrder)) {
-          boolean emailDeliveryStatus = testResultsDeliveryService.emailTestResults(patientLink);
-          deliveryStatuses.add(emailDeliveryStatus);
-        }
-      }
-
-      boolean deliveryStatus =
-          deliveryStatuses.isEmpty() || deliveryStatuses.stream().anyMatch(status -> status);
-      return new AddTestResultResponse(savedOrder, deliveryStatus);
+      return savedOrder;
     } finally {
       unlockOrder(order.getInternalId());
     }
   }
 
+  private void editMultiplexResults(TestOrder order, MultiplexTestResult newResults) {
+    Set<Result> resultsToUpdate = order.getResultSet();
```

Comment:
Do you know what the behavior here is if this is the first time we're editing the queue item? I would assume that `order.getResultSet()` would return an empty set in that case, but at this point I have no idea what weirdness Hibernate will generate 🤷